backend/server.js

```diff
@@ -0,0 +1,33 @@
+import fetch from "node-fetch";
+import express from 'express';
+import cors from 'cors';
+
+const PORT = 5000;
+const app = express();
+
+app.use(cors());
+const corsOptions = {
+    origin: "http://localhost:3000"
+};
+
+const riotAPI = "https://euw1.api.riotgames.com/lol"
+
+
+// This function runs if the http://localhost:5000/getData endpoint
+// is requested with a GET request
+app.get('/getData', cors(corsOptions), async (req, res) => {
+    const fetchOptions = {
+        method: 'GET',
+        headers:{
+            "X-Riot-Token": "RGAPI-a6e7e611-b5ed-41b4-84d2-18029e28e0d0"
```

Comment:
Tokens should not be in the source code (optimally), we should expolore the possibility of using .env variables of some sorts. This is fine for now, but wont be acceptable in a production environment.